src/stats/JavaToRConversion.java

```diff
@@ -42,6 +52,7 @@ public static RCode JavaToR(double[][] aMatrix){
         //These lines change the Java structure into the correct format for R
         code.addRCode("matrix <- (t(data))");
         code.addRCode("provideDimnames(matrix)");
+        //code.addRCode("matrix[matrix==-1] <- NA");
```

Comment:
Why's this commented out? Again, it should either be removed, or there should be a comment explaining why it might need to be uncommented, and in what scenario.